slither: The make cmd to run slither stopped working

Running `make analyze-slither` errors with:
```
Traceback (most recent call last):
  File "/Users/pascal/.pyenv/versions/sol/lib/python3.8/site-packages/slither/__main__.py", line 744, in main_impl
    ) = process_all(filename, args, detector_classes, printer_classes)
  File "/Users/pascal/.pyenv/versions/sol/lib/python3.8/site-packages/slither/__main__.py", line 76, in process_all
    compilations = compile_all(target, **vars(args))
  File "/Users/pascal/.pyenv/versions/sol/lib/python3.8/site-packages/crytic_compile/crytic_compile.py", line 637, in compile_all
    compilations.append(CryticCompile(target, **kwargs))
  File "/Users/pascal/.pyenv/versions/sol/lib/python3.8/site-packages/crytic_compile/crytic_compile.py", line 117, in __init__
    self._compile(**kwargs)
  File "/Users/pascal/.pyenv/versions/sol/lib/python3.8/site-packages/crytic_compile/crytic_compile.py", line 548, in _compile
    self._platform.compile(self, **kwargs)
  File "/Users/pascal/.pyenv/versions/sol/lib/python3.8/site-packages/crytic_compile/platform/foundry.py", line 143, in compile
    compilation_unit.srcmaps_init[contract_name] = target_loaded["bytecode"][
KeyError: 'sourceMap'
None
Error in .
Traceback (most recent call last):
  File "/Users/pascal/.pyenv/versions/sol/lib/python3.8/site-packages/slither/__main__.py", line 744, in main_impl
    ) = process_all(filename, args, detector_classes, printer_classes)
  File "/Users/pascal/.pyenv/versions/sol/lib/python3.8/site-packages/slither/__main__.py", line 76, in process_all
    compilations = compile_all(target, **vars(args))
  File "/Users/pascal/.pyenv/versions/sol/lib/python3.8/site-packages/crytic_compile/crytic_compile.py", line 637, in compile_all
    compilations.append(CryticCompile(target, **kwargs))
  File "/Users/pascal/.pyenv/versions/sol/lib/python3.8/site-packages/crytic_compile/crytic_compile.py", line 117, in __init__
    self._compile(**kwargs)
  File "/Users/pascal/.pyenv/versions/sol/lib/python3.8/site-packages/crytic_compile/crytic_compile.py", line 548, in _compile
    self._platform.compile(self, **kwargs)
  File "/Users/pascal/.pyenv/versions/sol/lib/python3.8/site-packages/crytic_compile/platform/foundry.py", line 143, in compile
    compilation_unit.srcmaps_init[contract_name] = target_loaded["bytecode"][
KeyError: 'sourceMap'

make: *** [analyze-slither] Error 255
```

--- a/Makefile
+++ b/Makefile
@@ -54,8 +54,14 @@ testE2e: ## Rune e2e test suite
 # Static Analyzers
 
 .PHONY: analyze-slither
-analyze-slither: ## Run slither analyzer against project
-	@slither .
+analyze-slither: ## Run slither analyzer against project (requires solc-select)
+	@forge build --extra-output abi --extra-output userdoc --extra-output devdoc --extra-output evm.methodIdentifiers
+	@solc-select use 0.8.13
+	@slither --ignore-compile src/common   || \
+	slither --ignore-compile src/factories || \
+	slither --ignore-compile src/generated || \
+	slither --ignore-compile src/modules   || \
+	slither --ignore-compile src/proposal
 
 .PHONY: analyze-c4udit
 analyze-c4udit: ## Run c4udit analyzer against project

